### PR TITLE
Added flag `-page-Id` for check sub-command of bbolt CLI.

### PR DESCRIPTION
**Pull request by ishan16696**
Part of this feature: https://github.com/etcd-io/bbolt/issues/580

### CODE REVIEW

File: tx_check.go [hunk 1]
```diff
@@ -250,19 +250,20 @@ func verifyKeyOrder(pgId common.Pgid, pageType string, index int, key []byte, pr
 
 type checkConfig struct {
 	kvStringer KVStringer
-	pageId     uint
+	pageId     uint64
```
Comment (ahrtr) [inline tx_check.go:253]: Please change this in a separate PR, so that we can merge right away

File: cmd/bbolt/main_test.go [hunk 2]
```diff
@@ -93,6 +93,20 @@ func TestCheckCommand_Run(t *testing.T) {
...
 }
 
+func TestCheckCommandWithPageID_Run(t *testing.T) {
+	db := btesting.MustCreateDB(t)
+	db.Close()
+
+	defer requireDBNoChange(t, dbData(t, db.Path()), db.Path())
+
+	m := NewMain()
+	err := m.Run("check", "-page-Id", "0", db.Path())
```
Comment (ahrtr) [inline cmd/bbolt/main_test.go:103]: The parameter "0" doesn't make sense, as the default value is 0, it has the same behaviour as no parameter being provided at all. We should populate some sample data, and find out a branch page ID and at least a leaf page ID, and run `bbolt check` on the page IDs.

**Comment by ahrtr** [comment]
I think we can directly migrate the `bbolt check` command to cobra style command directly. It doesn't break any existing user experience, so it's fine to migrate it firstly, afterwards, we can continue to add option `--pageId`.

Refer to: https://github.com/etcd-io/bbolt/issues/472

File: tx_check.go (hunk 1 above)
Comment (ishan16696) [reply tx_check.go:253]: Opened a PR: https://github.com/etcd-io/bbolt/pull/699

**Comment by ahrtr** [comment]
@ishan16696 Are you still working on this PR? Please resolve the two comments above. Please let me know if you need others' help.

**Comment by ishan16696** [comment]
> Are you still working on this PR? Please resolve the two comments above. Please let me know if you need others' help.

I will only be able to work on this PR during this weekend.

**Pull request closed without merging**